docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3.7"
+
+services:
+  caddy:
+    image: caddy:2.2.1-alpine
+    environment:
+      URL:
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - ./configurations/Caddyfile:/etc/caddy/Caddyfile:ro
+  postgres: # Must be identical to the .env POSTGRES_HOST variable
+    image: postgres:12.1
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER:?Required}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:?Required}
+      POSTGRES_DB: ${POSTGRES_DB:?Required}
+      DB_HOST: ${POSTGRES_HOST:?Required}
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+  postfix: # temporary
+    image: arugifa/simplelogin-postfix:3.1.0-6
+    ports:
+      - "25:25"
+    environment:
+      POSTFIX_FQDN: ${EMAIL_DOMAIN}
+      ALIASES_DEFAULT_DOMAIN: ${EMAIL_DOMAIN}
+      DB_NAME: ${POSTGRES_DB}
+      DB_HOST: ${POSTGRES_HOST}
+      DB_USER: ${POSTGRES_USER}
+      DB_PASSWORD: ${POSTGRES_PASSWORD}
+      LETSENCRYPT_EMAIL: admin@sl.com
```

Comment:
`sl.com` is registered and has a valid MX record. It might be confused to the domain holder if users of SL forget to change this address since all cert expiration notification will be sent to that domain. I recommend using a .test domain or other invalid LTDs to avoid this...

Or we can even just put this in env file and leave it blank as default, I believe let's encrypt can issue cert w/o having an email address.